.ci-build.yml

```diff
@@ -212,14 +181,20 @@ before_scripts:
         mkdir -p build
         cd build
         cmake -DLIBSWORD_LIBRARY_TYPE="Shared" -DSYSCONF_INSTALL_DIR='/etc' -DSWORD_BUILD_TESTS="Yes" -DSWORD_BINDINGS="Python" -DCMAKE_INSTALL_PREFIX=/usr -DLIB_INSTALL_DIR=/usr/lib/x86_64-linux-gnu ..
-        make -j2
+        make -j$(nproc)
         make install
     fi
 
 build_scripts:
   - |
-    mkdir -p build
-    cd build
+    set -ex
     export PKG_CONFIG_PATH=/usr/lib/pkgconfig
-    cmake -DCMAKE_INSTALL_PREFIX=/usr -DGTKHTML=ON -DCMAKE_MAKE_PROGRAM=/usr/bin/make ..
-    make -j2
+    cmake -B build -DCMAKE_INSTALL_PREFIX=/usr -DGTKHTML=ON -DCMAKE_MAKE_PROGRAM=/usr/bin/make .
+    make -C build -j$(nproc)
+    make -C build mhelp package_source
+    echo "::set-output name=artifact::$(ls build/*.tar.gz)"
+    #if [ "${DISTRO_NAME}" == "fedora" ]; then
```

Comment:
If these cases got taken care of another way, maybe drop the commented code before merger?

Reply:
Aaa, from one of your list emails...

> All that remains is adding the Linux artifact stashing and releasing on tag events. That's just a matter of about four more steps in Github [...]

I'm guessing that might be related. Carry on!

Reply:
I skipped generating these binary artifacts because I couldn't guard them. But I should be able to generate directly with calls to "cpack -G" so I'm testing that now.

Reply:
OK, this is now working as expected.